src/evaluation/liveRec/buildIdentificationBBS.m

```diff
@@ -0,0 +1,14 @@
+function bbs = buildIdentificationBBS(sim,idModels,labels,onOffsets)
+
+bbs = BlackboardSystem(1);
+bbs.setRobotConnect(sim);
+bbs.setDataConnect('AuditoryFrontEndKS');
+for ii = 1 : numel( idModels )
+    idKss{ii} = bbs.createKS('IdentityKS', {idModels(ii).name, idModels(ii).dir});
+    idKss{ii}.setInvocationFrequency(50);

```

Comment:
I don't know much about the BBS, but there should be little benefit in having invocation frequency > min( [AFE Proc FsOutHz] ). This should default to the 100Hz framing of the AFE for most procs.